circle.yml

```diff
@@ -36,10 +36,14 @@ jobs:
 #          name: Sonarcloud upload
 #          command: make sonarqube
       - run:
-          name: Build testapp APK
+          name: Build Test Application APK
           command: |
             ./gradlew accessToken
             ./gradlew app:assembleDebug
+      - run:
```

Comment:
I guess we want this step to be after `Build release to test ProGuard rules` one. No need to run it if `Build release to test ProGuard rules` fails right?